database-redis/index.js

```diff
@@ -0,0 +1,14 @@
+const redis = require("redis");
+
+const client = redis.createClient(5002, "127.0.0.1");
+
+client.on("connect", function() {
+  console.log("Redis client connected");
+});
+
+client.on("error", function(err) {
+  console.log("Something went wrong " + err);
+});
+
+
+module.exports = client;
```

Comment:
I noticed the warning from Github about this line (there's a red ↩️🚫 at the end of line 14 that says "No newline at end of file" when you hover over it). 

I found [this](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file) SO post -- apparently it's better to add a blank line at the end of files, and some other languages like C and C++ require it.

Reply:
Cool, good to know. Will add the extra line.